Meadowlark-js/.vscode/launch.json

```diff
@@ -9,7 +9,7 @@
       "args": ["offline", "start", "--printOutput", "--stage", "local", "--allowCache"],
       "port": 9229,
       "console": "integratedTerminal",
-      "cwd": "${workspaceFolder}/packages/meadowlark-aws-frontend",
+      "cwd": "${workspaceFolder}/services/meadowlark-aws-lambda",
```

Comment:
Looking for a way to distinguish front end pakages, the use of "services" folder parallel with "packages" for serverless projects is a common pattern.